public/gameInfo.php

```diff
@@ -101,14 +126,31 @@
                 Are you sure that you want to view this game?
                 <br/>
                 <br/>
-                <form id='consentform' action='/game/<?= $gameID ?>' style='float:left'>
-                    <input type='hidden' name='v' value='1'/>
-                    <input type='submit' value='Yes. I&apos;m an adult'/>
-                </form>
-                <form id='escapeform' action='/gameList.php' style='float:left; margin-left:16px'>
-                    <input type='hidden' name='c' value='<?= $consoleID ?>'/>
-                    <input type='submit' value='Not Interested'/>
-                </form>
+
+                <div class="flex flex-col sm:flex-row gap-4 sm:gap-2">
+                    <form id='escapeform' action='/gameList.php'>
+                        <input type='hidden' name='c' value='<?= $consoleID ?>'/>
+                        <input type='submit' class='leading-normal' value='No, get me out of here.'/>
+                    </form>
+
+                    <form id='consentform' action='/game/<?= $gameID ?>'>
+                        <input type='hidden' name='v' value='1'/>
+                        <input type='submit' class='leading-normal' value='Yes, I&apos;m an adult.'/>
```

Comment:
Google's [grammar suggestions](https://www.margieholdscourt.com/after-yes-and-no/#:~:text=Let's%20keep%20it%20simple%3A,the%20question%2C%20use%20a%20comma.) say that you should use a period unless you're echoing the question. 
`Yes, I want to view it` vs `Yes. I'm an adult.`

I think that a period is appropriate for all three of these lines.

Reply:
Sounds so strange and unnatural to me. Like a robot talking.

Reply:
The comma connects statements. "Yes, I'm an adult." is a confirmation to a question that wasn't asked. "Yes. I'm an adult." is one statement answering the question that was asked, and another statement providing additional context to the response.

Reply:
Grammar-wise, it makes sense. It's just that the period-induced pause makes the reply feel more mechanic/blunt and less "friendly", at least to me.

Some mature sites [do indeed](https://user-images.githubusercontent.com/22218549/219454967-9bc784a6-0137-444a-a2e2-cfd3f6d43d1b.png) use commas for dialogs like these. The majority, however, simply [drop](https://user-images.githubusercontent.com/22218549/219456371-aff7f922-f647-4dcf-8d70-c40f469523b9.png)
 the "yes/no" part altogether, or just don't use punctuation at all. I guess in the end it merely depends on what impression we want to leave in the user.

Reply:
I've adjusted this to follow "[Yes|No]. [phrase]".